setup.py

```diff
@@ -32,14 +32,20 @@
 
     provides=[],
     install_requires=[
-        'beautifulsoup4',
-        'beautifultable',
-        'click',
-        'colorama',
-        'pandas',
-        'requests',
-        'responses',
-        'validators',
+        'beautifulsoup4==4.6.0',
+        'beautifultable==0.5.0',
+        'click==6.7',
+        'coverage==4.5.1',
+        'coveralls==1.3.0',
+        'flake8==3.0.4',
+        'lxml==4.2.1',
+        'pytest==3.5.1',
+        'pytest-cov==2.5.1',
+        'python-dateutil==2.7.3',
+        'requests==2.18.4',
+        'requests-toolbelt==0.8.0',
```

Comment:
Are you sure we need `requests-toolbelt`?

Reply:
This will be installed as requests will be installed. So Ill change this